plotting/setup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+sudo apt-get update
+sudo apt-get install r-base
+Rscript R_scripts/install.R
+
```

Comment:
further, I'm a little wary of installing R onto a reviewer/repo cloner's machine; if this ends up part of the repro workflow, this may even be done silently (read: without reviewer consent). I'd personally feel a bit better if we installed and ran from the `build` or `lib` dir

Reply:
I'm ok with installing it from the build or lib dir. Who should set that up? I'll remove this entire file for now, whoever adds this elsewhere should make sure to also run the install.R script as well since it installs the R plotting library we use.

Reply:
Actually, it looks like installing without sudo apt-get is hard and unnecessary. ig we can just tell the reviewer that we plan on installing R [y/n] and they can uninstall later.

Reply:
I took a look at how setup works without using apt-get. It’s not pretty. I think in this case we’ll just need to tell the reviewer that this will download R and give them a command to uninstall it later

Reply:
We definitely need to prompt before using sudo. Or, even better have the sudo steps be part of our explicit installation instructions and not even in a script. (like what we did for cmake, etc.)